packages/@aws-cdk/core/lib/cfn-fn.ts

```diff
@@ -172,6 +172,18 @@ export class Fn {
     return new FnFindInMap(mapName, topLevelKey, secondLevelKey).toString();
   }
 
+  /**
+   * The intrinsic function Fn::Transform specifies a macro to perform custom processing on part of a stack template.
+   * Macros enable you to perform custom processing on templates, from simple actions like find-and-replace operations
+   * to extensive transformations of entire templates.
```

Comment:
This comment should be more about the CDK than quoting the CFN docs for `Fn::Transform`.

```ts
/**
 * Creates a token representing the Fn::Transform expression
...
*/
```

Reply:
Also, you would be great to have a `@see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-transform.html` there.